modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -55,55 +85,30 @@ spec:
 +
 [NOTE]
 ====
-Using the *preview-1.1* channel ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.
+Using the *preview-{sandboxed-containers-version}* channel ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.
```

Comment:
> channel ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.

I don't think we add much information here in this statement, especially since we are trying to get their attention to read it.

Reply:
@zanetworker - this was in the original docs. Should I remove it?